README.md

```diff
@@ -13,7 +13,7 @@ Run `black` on python code blocks in documentation files.
 ## usage
 
 `blacken-docs` provides a single executable (`blacken-docs`) which will modify
-`.rst` / `.md` files in place.
+`.rst` / `.md` / `.tex` files in place.
```

Comment:
we'll want to update https://github.com/asottile/blacken-docs/blob/f9166fe61ca1e1c6def8be3ea61ac91d798c4336/.pre-commit-hooks.yaml#L7